packages/ui-kit/src/themes/_common.scss

```diff
@@ -45,4 +45,34 @@
 
   /* Shadows */
   --propel-shadow-sm: 0px 2px 4px 0px rgba(17, 19, 34, 0.1);
+
+  /* Color Blue (Brand)*/
+  --color-blue-950: #102A56;
+  --color-blue-900: #194185;
+  --color-blue-800: #1849A9;
+  --color-blue-700: #175CD3;
+  --color-blue-600: #1570EF;
+  --color-blue-500: #2E90FA;
+  --color-blue-400: #53B1FD;
+  --color-blue-300: #84CAFF;
+  --color-blue-200: #B2DDFF;
+  --color-blue-100: #D1E9FF;
+  --color-blue-50: #EFF8FF;
+  --color-blue-25: #F5FAFF;
 }
+
+:export {
```

Comment:
With the [current architecture](https://linear.app/propel/issue/PRO-2107/add-theme-support), we don't have to export tokens from sass. If you add new tokens to [packages/ui-kit/src/themes/theme.types.ts](https://github.com/propeldata/ui-kit/blob/main/packages/ui-kit/src/themes/theme.types.ts) and [packages/ui-kit/src/themes/themeTokens.ts](https://github.com/propeldata/ui-kit/blob/main/packages/ui-kit/src/themes/themeTokens.ts), you will be able to access the token values in `theme` prop of `const { theme, chartConfig } = useSetupTheme<'pie' | 'doughnut'>({ componentContainer, baseTheme })`

Reply:
Currently, you have
```tsx
const defaultChartColorPalette: string[] = [
  componentStyles.color_blue_800,
  componentStyles.color_blue_700,
  componentStyles.color_blue_600,
  componentStyles.color_blue_500,
  componentStyles.color_blue_400,
  componentStyles.color_blue_300,
  componentStyles.color_blue_200,
  componentStyles.color_blue_100,
  componentStyles.color_blue_50,
  componentStyles.color_blue_25
] 
```

You may get the same with the current architecture:
```tsx
const { theme, chartConfig } = useSetupTheme<'pie' | 'doughnut'>({ componentContainer, baseTheme })

const defaultChartColorPalette = useMemo(() => [theme.colorBlue800, theme.colorBlue700, ...], [theme])
```